src/main.rs

```diff
@@ -37,6 +37,22 @@ fn main() -> anyhow::Result<()> {
             .info(CompileTimeInformation::default())
             .print::<Markdown>();
         std::process::exit(0);
+    } else if let Some(id) = args.explain.as_deref() {
+        let queries = query::SemverQuery::all_queries();
+        let query = queries.get(id).ok_or_else(|| {
+            let ids = queries.keys().cloned().collect::<Vec<_>>();
+            anyhow::format_err!(
+                "Unknown id `{}`, available id's:\n  {}",
+                id,
+                ids.join("\n  ")
+            )
+        })?;
+        println!("{}", query.description);
+        if let Some(link) = &query.reference_link {
+            println!();
+            println!("See also {}", link);
+        }
```

Comment:
I'm assuming you've thought about showing the implementation link, and have decided it isn't worth it? No strong feelings one way or the other on my end, just wanted to get your thoughts.

I had originally included it in the error message information in case someone felt an error was false-positive, to make it possible to review the query and/or report an issue more easily.

Reply:
I didn't think about it but I also don't think it should go in here.  We should keep the scope of what is shown to be fairly narrow and as pertinent as possible.